### PR TITLE
Add Favicon, OG tags and twitter tags

### PR DESCRIPTION
**Pull request by wseagar**
**OG Image (for social sharing)**
image

**Favicon**
image

### CODE REVIEW

File: src/components/SEO.tsx [hunk 1]
```diff
@@ -25,6 +28,53 @@ const SEO = (): JSX.Element => {
         name="description"
       />
       <title>vaxxed.as | {translate("header.Scan your NZ COVIDpass")}</title>
+      <link
```
Comment (rafcontreras) [inline src/components/SEO.tsx]: No need to add the favicon, webmanifest, tile, and theme colours here. 

Please take a look at the `gatsby-config.js` file, there's where the magic happens.

Favicons are automatically generated as well. Just overwrite the SVG file at `./src/images/icon.svg`

File: src/components/SEO.tsx [hunk 2]
```diff
@@ -25,6 +28,53 @@ const SEO = (): JSX.Element => {
...
+        rel="icon"
+        sizes="16x16"
+        type="image/png"
+      />
+      <link href="/site.webmanifest" rel="manifest" />
+      <link color="#5bbad5" href="/safari-pinned-tab.svg" rel="mask-icon" />
+      <meta content="#da532c" name="msapplication-TileColor" />
+      <meta content="#ffffff" name="theme-color" />
+
+      <meta content="website" property="og:type" />
```
Comment (rafcontreras) [inline src/components/SEO.tsx:28]: From here all good.

File: static/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (rafcontreras) [inline static/site.webmanifest]: This is generated at the `gatsby-config.js` file.

File: static/safari-pinned-tab.svg [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (rafcontreras) [inline static/safari-pinned-tab.svg]: This looks like a black square.

File: static/browserconfig.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (rafcontreras) [inline static/browserconfig.xml]: This is automatically generated at `gatsby-config.js`, please remove

**Review by rafcontreras — CHANGES_REQUESTED** [review]
Please remove favicon files

File: src/components/SEO.tsx (hunk 1 above)
Comment (wseagar) [reply src/components/SEO.tsx]: Ah yep sweet, never used gatsby before I usually just do meta tags myself

**Comment by wseagar** [comment]
Removed all the favicon stuff and just left the OG image and the og tags @rafcontreras

**Comment by rafcontreras** [comment]
@wseagar looks like you didn't add the SVG image for the new favicon

**Review by rafcontreras — APPROVED** [review]

**Pull request merged**